plugins/clusters.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: clusters
+spec:
+  version: "v0.1.1"
+  platforms:
+  - uri: https://github.com/u2takey/kubectl-clusters/archive/v0.1.1.tar.gz
+    sha256: 369dcf9c83f38b0068a5176b802285bfe4cfda60b7e0b4c65841e5c354fd8bc7
+    bin: kubectl-clusters
+    files:
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
```

Comment:
Please include your `LICENSE` file and install that along with your binary file.

Reply:
LICENSE added